Assets/Unity3DTiles/Unity3DTileset.cs

```diff
@@ -86,25 +87,29 @@ public double TimeSinceLoad
             }
         }
 
-        public Unity3DTileset(Unity3DTilesetOptions options, MonoBehaviour behaviour, RequestManager requestManager)
+        public Unity3DTileset(Unity3DTilesetOptions tilesetOptions, AbstractTilesetBehaviour behaviour, RequestManager requestManager, LRUCache<Unity3DTile> cache = null)
         {
-            this.Options = options;
+            this.TilesetOptions = tilesetOptions;
             this.Behaviour = behaviour;
             this.RequestManager = requestManager;
-            this.traversal = new Unity3DTilesetTraversal(this);
-            this.LRUContent = new LRUCache<Unity3DTile>();
+            this.traversal = new Unity3DTilesetTraversal(this, behaviour.SceneOptions);
+            this.LRUContent = cache ?? new LRUCache<Unity3DTile>();
             this.DeepestDepth = 0;
+            this.TilesetTransform = new GameObject("transform").transform;
```

Comment:
this is important as it will enable raycast hits to walk up the scene hierarchy and figure out which tileset was hit

Reply:
Axing `TilesetTransform` as it is already handled here: https://github.com/NASA-AMMOS/Unity3DTiles/pull/5/files#r330751714

Reply:
OK but then what is the plan to know what tileset was hit by a raycast when multiple tilesets are loaded?  If  you have the root of each tileset parented to its own GameObject where the name of that GameObject is the name of the tileset then, given a RaycastHit anywhere in that tileset tree, it should be possible to walk up the tree and eventually get to that root GameObject and thus get the name of the tileset.

There are other and maybe better ways to solve the problem, for example, by adding a small Behaviour onto every GameObject in the whole tileset hierarchy, where that Behaviour contains a reference to the tileset.  One upside of doing it that way would be that the RaycastHit could be resolved to the tileset in constant time, no tree walk required (though I doubt that the tileset hierarchy tree is ever arbitrarily tall, so that is only a theoretical concern, but the code would be a bit simpler for the client anyway).  One downside is that you'll have to figure out where you can reliably add that Behaviour onto all the tileset GameObjects, given that if I'm reading the existing code right, I think they are actually managed deep in the GLTF code.